pkg/server/adminHandler.go

```diff
@@ -4,22 +4,34 @@
 package server
 
 import (
+	"bytes"
 	"crypto/x509"
 	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"net/http"
 	"strings"
 
+	"github.com/golang/protobuf/jsonpb"
 	"github.com/gorilla/mux"
-	"github.com/lf-edge/eve/api/go/config"
-	"github.com/satori/go.uuid"
 	"github.com/lf-edge/adam/pkg/driver"
 	ax "github.com/lf-edge/adam/pkg/x509"
+	"github.com/lf-edge/eve/api/go/config"
+	"github.com/lf-edge/eve/api/go/info"
+	"github.com/lf-edge/eve/api/go/logs"
+	uuid "github.com/satori/go.uuid"
+)
+
+const (
+	StreamHeader = "X-Stream"
+	StreamValue  = "true"
```

Comment:
LOL -- I love how this constant reads "extreme"! ;-)

Reply:
Hohoho! I didn't think of that!